r-package/survlearners/R/surv_rl_grf.R

```diff
@@ -47,38 +47,38 @@
 #' Y <- pmin(failure.time, censor.time)
 #' D <- as.integer(failure.time <= censor.time)
 #'
-#' rgrf_fit = rgrf(X, W, Y, D, times, p_hat = 0.5)
-#' rgrf_cate = predict(rgrf_fit, X)
+#' surv_rl_grf_fit = surv_rl_grf(X, W, Y, D, times, p_hat = 0.5)
+#' cate = predict(surv_rl_grf_fit)
```

Comment:
ideally add a 2nd line for each
cate.test = predict(surv_rl_grf_fit, X.test)
to make sure this too runs without problem before more tests are added

Reply:
Good point, will do! thanks